test-app-async/src/test/resources/application-test.yml

```diff
@@ -14,24 +14,34 @@ hmpps.sqs:
       queueName: ${random.uuid}
       subscribeTopicId: inboundtopic
       subscribeFilter: '{"eventType":[ "OFFENDER_MOVEMENT-RECEPTION", "OFFENDER_MOVEMENT-DISCHARGE", "test.type"] }'
+      asyncDlqClient: true
+      asyncQueueClient: true
```

Comment:
I'm wondering if the default should be true for this version (seeing as it's already a breaking change).